packages/react-components/react-combobox/src/components/Combobox/renderCombobox.tsx

```diff
@@ -24,7 +23,7 @@ export const renderCombobox_unstable = (state: ComboboxState, contextValues: Com
           (state.inlinePopup ? (
             <slots.listbox {...slotProps.listbox} />
           ) : (
-            <Portal>
+            <Portal {...slotProps.portal}>
```

Comment:
I wanted to use slots.portal here, but Portal is a pure function and not a ForwardRef component, so I get this error:

JSX element type 'slots.portal' does not have any construct or call signatures.ts(2604)